docs/knowledgebase/integrate/subkey.md

```diff
@@ -258,23 +249,21 @@ Recall the SS58 address from the same seed phrase,
 address.
 
 ```bash
-$ subkey inspect "5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0"
+$ subkey inspect-key --uri "5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0"
 Public Key URI `5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0` is account:
-  Network ID/version: substrate
+  Network ID/version: polkadot
   Public key (hex):   0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
   Account ID:         0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
-  SS58 Address:       5CCrqJffST8uh8RLn2sFJwg1JWs4HkKL88xmdZRdnHt8GDyA
+  SS58 Address:       199ydvjJEQP8fRrjfvFT6WAA8rhz3sUCdhFnrQzLNueSU5G
 ```
 
-Note that the two addresses here match. This is not the case in hard key derivation.
```

Comment:
@shawntabrizi, @joepetrowski - do you  know why this isn't working?

Reply:
There is no `--uri` flag:

```
subkey inspect-key 5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0 
 
Public Key URI `5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0` is account:
  Network ID/version: substrate
  Public key (hex):   0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  Account ID:         0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  SS58 Address:       5CCrqJffST8uh8RLn2sFJwg1JWs4HkKL88xmdZRdnHt8GDyA
```

Reply:
That does not work for me:
```shell
substrate.dev: subkey --version
subkey 2.0.0-rc6
substrate.dev: subkey inspect-key 5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0
error: Found argument '5CXFinBHRrArHzmC6iYVHSSgY1wMQEdL2AiL6RmSEsFvWezd/joe/polkadot/0' which wasn't expected, or isn't valid in this context

USAGE:
    subkey inspect-key [FLAGS] [OPTIONS]

For more information try --help
```

Reply:
Seems the default changed from Substrate to Polkadot. Matches just fine on the old Subkey:
```
subkey -n polkadot inspect 5CCrqJffST8uh8RLn2sFJwg1JWs4HkKL88xmdZRdnHt8GDyA
Public Key URI `5CCrqJffST8uh8RLn2sFJwg1JWs4HkKL88xmdZRdnHt8GDyA` is account:
  Network ID/version: polkadot
  Public key (hex):   0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  Account ID:         0x06382e496cb8b1664501bbfcc8205c7b1ddc2402e32ef0479b18abdf326ca342
  SS58 Address:       199ydvjJEQP8fRrjfvFT6WAA8rhz3sUCdhFnrQzLNueSU5G
```